Makefile

```diff
@@ -1456,6 +1503,7 @@ mpas_main: $(MAIN_DEPS)
 	@echo $(PARALLEL_MESSAGE)
 	@echo $(MPI_F08_MESSAGE)
 	@echo $(PAPI_MESSAGE)
+	@echo $(MUSICA_MESSAGE)
```

Comment:
We could move this message after the `OPENACC_MESSAGE` on line 1510 rather than placing it between the PAPI and TAU messages (PAPI and TAU are for performance measurement and profiling, and so serve related roles).